Bottom: e6c79c466c36403b069591ba071b57ece4f37e51
Top:    c9fc6acc78eda733d9fe3406235acba7a5187902
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-27 14:26:55 +0100

Refresh of add-simple-step-exporter-for

---

diff --git a/src/Makefile.am b/src/Makefile.am
index b463979..d1eb83e 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -279,6 +279,7 @@ EXTRA_DIST= \
 	$(srcdir)/hid/png/hid.conf \
 	$(srcdir)/hid/nelma/hid.conf \
 	$(srcdir)/hid/ps/hid.conf \
+	$(srcdir)/hid/step/hid.conf \
 	gpcb-menu.res.in \
 	pcb-menu.res.in \
 	gpcb-menu.res.h \
